src/download.js

```diff
@@ -0,0 +1,66 @@
+const process = require('process');
+const homedir = require('os').homedir();
+const { createWriteStream, chmodSync } = require('fs');
+const { pathExists, ensureDir } = require('fs-extra');
+const { dirname } = require('path');
+const axios = require('axios');
+
+const REPOSITORY = 'OpenZeppelin/openzeppelin-gsn-helpers';
+const VERSION = 'v0.1.4';
+const BINARY = 'gsn-relay';
+const PATH = `${homedir}/.gsn/${BINARY}-${VERSION}`;
```

Comment:
I think it's a bit intrusive to install in the home directory.

What do you think about downloading to the module's installation directory? It won't cache across different installations though...

```suggestion
const PATH = `${__dirname}/${BINARY}-${VERSION}`;
```

Alternatively we could use this: [`env-paths`](https://github.com/sindresorhus/env-paths).

Reply:
Hmm I wanted to install to home precisely to cache between runs. That said, I liked `env-paths`, will change this to use it!